Nos/Models/CoreData/Author+CoreDataClass.swift

```diff
@@ -41,10 +41,41 @@ import Logger
         return PublicKey(hex: hex)
     }
 
+    var hasNIP05: Bool {
+        if let nip05, !nip05.isEmpty {
+            return true
+        } else {
+            return false
+        }
+    }
+
     var hasNosNIP05: Bool {
         nip05?.hasSuffix("@nos.social") == true
     }
 
+    var hasUNS: Bool {
+        if let uns, !uns.isEmpty {
+            return true
+        } else {
+            return false
+        }
+    }
+
+    var nip05Parts: (username: String, domain: String)? {
```

Comment:
@martindsq maybe I could refactor `WizardSheetDescriptionText` to use this? Haven't looked at it yet, but I used this in the updated NIP05View to show the username in Color.primaryTxt and the domain in Color.secondaryTxt.

Reply:
What I solved in the wizard was being able to show localized text with people names in it, where the people names would be displayed in a different color. That could be used inside KnownFollowersView.

Reply:
@martindsq ahhh, I see what you're saying now. I'm struggling to come up with a good rename for it. Basically I want to convey what's happening with the `markdown:tint:` part. Something like `MulticolorText(markdown:replacingStronglyEmphasizedWithColor:)`. But that seems a bit long...

Reply:
Maybe I just break those into multiple parameters. Something like this? 
`init(markdown: String, replacing: InlinePresentationIntent, with: Color)`

Reply:
Or I don't need to reuse the type at all; I can just copy the attributedString code. I'll do that. Thanks!